AP-protocol.md

```diff
@@ -188,40 +188,40 @@ pub struct MessageData { // Only part fragmentized
 }
 ```
 
-### NACK
-If an error occurs that a NACK is sent. A NACK can be of type:
+### Ack
+
+If a drone receives a Message and can forward it to the next hop, it also sends an Ack to the client.
+
+```rust
+pub struct Ack{
+	fragment_index: u64,
+	time_received: std::time::Instant
+}
+```
```

Comment:
I think that ACK are sent when the message is completed at the receiving end, not at every jump of every fragment.